lib/helper/validator.helper.js

```diff
@@ -0,0 +1,49 @@
+const Joi = require('joi');
+
+const configSchema = Joi.object({
+  debug: Joi.boolean().optional(),
+  disabled: Joi.boolean().optional(),
+  cache: Joi.boolean().optional(),
+  tokenLocation: Joi.string().optional(),
+  source: Joi.string()
+    .valid('authzv2', 'meauthzv2', 'externalAuthz')
```

Comment:
For backwards compatibility we also have `meauthz` which is an alias for `meauthzv2` (not documented as it might be removed in the future).